packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -867,6 +869,11 @@ const _BaseInput: React.ForwardRefRenderFunction<BladeElementRef, BaseInputProps
   const isReactNative = getPlatformType() === 'react-native';
   const defaultRole = hasPopup ? 'combobox' : undefined;
 
+  const inputGroupProps = useInputGroupContext();
+  const isWithinInputGroup = Object.keys(inputGroupProps).length > 0;
+  const _isDisabled = inputGroupProps.isDisabled || isDisabled;
```

Comment:
```suggestion
  const _isDisabled = inputGroupProps.isDisabled ?? isDisabled;
```

When InputGroup is disabled, we should disable all inputs inside right? With `||` I am assuming it will end up prioritising individual isDisabled when inputGroupProps.isDisabled is set to false

Reply:
Done